install-and-configure/advanced-configuration/gpu.md

```diff
@@ -0,0 +1,234 @@
+# NVIDIA GPU Monitoring Configurations
+
+## Monitoring GPU utilization
+
+In order for Kubecost to understand a container's GPU utilization percentage, Kubecost depends on metrics being available from NVIDIA [DCGM Exporter](https://github.com/NVIDIA/dcgm-exporter). Although Kubecost ships by default with GPU metrics being enabled, since DCGM Exporter is the provider of those metrics is a required component when GPU monitoring is used with Kubecost and must be installed if it is not already. In many cases, DCGM Exporter may already be installed in your cluster, for example if you currently monitor NVIDIA GPUs with other software. Follow the below instructions to install and configure DCGM Exporter on each of your GPU-enabled clusters if it is not installed.
```

Comment:
what DCGM exporter versions do we support? I feel if we aren't going to install on behalf of users we should be presecriptive here.

Reply:
While I have only tested the latest version (and I specify in the GKE section the version plus that of GKE), I have looked at the metrics being emitted by DCGM Exporter for the past over one year and it appears to have the one we need. I can state this version higher up here though.